packages/product-react/src/components/List/List.test.tsx

```diff
@@ -0,0 +1,46 @@
+import Enzyme, { configure, mount, ReactWrapper } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { List } from './List';
+
+configure({ adapter: new Adapter() });
+
+describe('List', () => {
+  let component: ReactWrapper<any>;
+
+  beforeEach(() => {
+    component = mount(<List items={[{}]} />);
+  });
+
+  test('should match snapshot and styles', () => {
+    expect(
+      mount(<List items={[{ header: 'header', content: 'content' }]} />),
+    ).toMatchSnapshot();
+  });
+
+  describe('Functions', () => {
+    it('onFocus', () => {
+      component = Enzyme.mount(
+        <List
+          items={[
+            {
+              header: 'Sammy Lawson',
+              content: 'Product Designer',
+              icon: 'laalalalaaal',
+            },
+            {
+              header: 'Arnold Schwarzenegger',
+              content: 'Product Designer',
+              icon: 'laalalalaaal',
+            },
+            {
+              header: 'Michael Jordan',
+              content: 'Product Designer',
+              icon: 'laalalalaaallaalalalaaal',
+            },
+          ]}
+        />,
```

Comment:
It looks like this test doesn't assert anything

Reply:
oo   your right and also don't even need this  test, there is no focus my bad